db/data/regions/IL/en.yml

```diff
@@ -5,4 +5,5 @@ en:
       addresses:
         zip:
           errors:
-            invalid: Enter a valid 7-digit postal code for Israel
+            invalid_instructional: Enter a valid 7-digit postal code for Israel
```

Comment:
Do we need to specify 7-digit if a valid Israeli postal code will always have 7 digits?

Reply:
Just leaving the `instructional` message the same as the `invalid` message.